src/containers/shared/components/Transaction/EscrowCreate/test/EscrowCreateTableDetail.test.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { I18nextProvider } from 'react-i18next'
+import { BrowserRouter } from 'react-router-dom'
+import { mount } from 'enzyme'
+import { TableDetail } from '../TableDetail'
+import i18n from '../../../../../../i18nTestConfig'
+import summarize from '../../../../../../rippled/lib/txSummary'
+import mockEscrowCreate from './mock_data/EscrowCreate.json'
+
+const createWrapper = (tx: any) =>
+  mount(
+    <BrowserRouter>
+      <I18nextProvider i18n={i18n}>
+        <TableDetail
+          instructions={summarize(tx, true).details.instructions}
+          type={tx.tx.TransactionType}
+        />
+      </I18nextProvider>
+    </BrowserRouter>,
+  )
+describe('EscrowCreateTableDetail', () => {
+  it('renders EscrowCreate without crashing', () => {
```

Comment:
nit: Update the `it` to not include `without crashing` the test is much more than that now :)